lib/openapi/revisions/fps-spec-1.5.0-20231130.json

```diff
@@ -0,0 +1,870 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "PhotonIQ Digital Fingerprint Service API",
+    "description": "API documentation for the PhotonIQ Digital Fingerprint Service",
+    "version": "1.5.0"
+  },
+  "servers": [
+    {
+      "url": "https://demofps.paas.macrometa.io"
+    }
+  ],
+  "components": {
+    "securitySchemes": {
+      "ApiKeyAuth": {
+        "type": "apiKey",
+        "in": "header",
+        "name": "Authorization"
+      }
+    },
+    "schemas": {
+      "GetVersionResponse": {
+        "type": "object",
+        "properties": {
+          "version": {
+            "type": "string"
+          }
+        }
+      },
+      "RecordVisitInformation": {
+        "type": "object",
+        "properties": {
+          "signals": {
+            "type": "object",
+            "description": "Each property (referred to as [signalName]) represents a unique signal.",
+            "additionalProperties": {
+              "type": "object",
+              "properties": {
+                "duration": {
+                  "type": "number",
+                  "description": "Duration, expected to be a number."
+                },
+                "error": {
+                  "type": "string"
+                },
+                "value": {
+                  "type": "string",
+                  "description": "Any value in string format"
+                }
+              }
+            }
+          },
+          "version": {
+            "type": "string"
+          }
+        }
+      },
+      "VisitResponse": {
+        "type": "object",
+        "properties": {
+          "visitorId": {
+            "type": "string"
+          },
+          "signature": {
+            "type": "string"
+          },
+          "visit": {
+            "type": "object",
+            "properties": {
+              "visitId": {
+                "type": "string"
+              },
+              "platformDetails": {
+                "$ref": "#/components/schemas/PlatformDetails"
+              },
+              "incognito": {
+                "type": "boolean"
+              },
+              "ip": {
+                "type": "string"
+              },
+              "url": {
+                "type": "string"
+              },
+              "visitorFound": {
+                "type": "boolean"
+              },
+              "confidence": {
+                "type": "object",
+                "properties": {
+                  "matchScore": {
+                    "type": "number"
+                  }
+                }
+              },
+              "timestamp": {
+                "type": "number",
+                "format": "date-time",
+                "description": "Visit timestamp"
```

Comment:
All descriptions should end with a period. Summaries should not have periods at the end.